orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/BuildArtifactFilter.groovy

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.orca.igor
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.env.Environment
+import org.springframework.stereotype.Component
+
+@Component
+class BuildArtifactFilter {
+  static final String MAX_ARTIFACTS_PROP = BuildArtifactFilter.simpleName + ".maxArtifacts"
```

Comment:
Any concern with the property name changing (other than needing to update our config)?

Not sure anyone in the community was actively tweaking these defaults.